spec/features/resource_manager_modifies_appointments_spec.rb

```diff
@@ -138,7 +137,7 @@ def and_there_is_a_bookable_slot_for_a_guider
     )
   end
 
-  def then_they_can_see_the_bookable_slot
+  def then_they_can_see_the_bookable_slot # rubocop:disable Metrics/AbcSize
```

Comment:
Just disable it globally at the top of the file instead of repeating everywhere, especially since actually disabling it once in the file disables the rule for the rest of the file from that line.